stanford_sites_helper.module

```diff
@@ -286,3 +286,11 @@ function stanford_sites_helper_stanford_sites_hosted() {
     return FALSE;
   }
 }
+
+/**
+ * Check to see if we are hosted on Acquia.
+ */
+function stanford_sites_helper_is_acquia_hosted() {
+  // If the AH_SITE_ENVIRONMENT environment variable is set, return TRUE. Otherwise, return FALSE.
```

Comment:
It won't return true, it will return the value of the environment variable.

Reply:
Right. Duh.